libs/core/src/provider/provider.registry.ts

```
@@ -856,17 +861,19 @@ export default class ProviderRegistry extends RegistryAbstract<
     this.graph.set(rec.provide, new Set());
     this.instances.set(rec.provide, rec.value);
   }
+
   private getWithParents<T>(token: Token<T>): T {
     let providers: ProviderRegistry = this
```

ESLint

[error] Unexpected aliasing of 'this' to local variable.

(@typescript-eslint/no-this-alias)